bigframes/session/validation.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import bigframes.constants
+
+ENGINE_ERROR_TEMPLATE = (
+    "write_engine='{write_engine}' is incompatible with engine='{engine}'. "
```

Comment:
Should we make it more helpful by suggesting what are valid combinations?

Reply:
I worry that will be hard to maintain. Perhaps we can move some of the combinations as constants, too?

Reply:
resolving the outdated comment